queries.sql

```diff
@@ -60,4 +60,30 @@ SELECT MAX(escape_attempts), neutered FROM animals GROUP BY neutered;
 
 SELECT MIN(weight_kg), MAX(weight_kg) FROM animals GROUP BY species;
 
-SELECT AVG(escape_attempts) FROM animals WHERE date_of_birth >= '1990-12-31' AND date_of_birth <= '2000-01-01' GROUP BY species;
\ No newline at end of file
+SELECT AVG(escape_attempts) FROM animals WHERE date_of_birth >= '1990-12-31' AND date_of_birth <= '2000-01-01' GROUP BY species;
+
+/* Project Day 3 */
+
+/*// Write queries (using JOIN) to answer the following questions:*/
+
+/*// 1 What animals belong to Melody Pond?*/
+SELECT animals.name FROM owners INNER JOIN animals ON animals.owner_id=owners.id AND owners.full_name = 'Melody Pond';
+
+/*// 2 List of all animals that are pokemon (their type is Pokemon).*/
+SELECT animals.name FROM species INNER JOIN animals ON animals.species_id=species.id AND species.name = 'Pokemon';
+
+/*// 3 List all owners and their animals, remember to include those that don't own any animal.*/
+SELECT animals.name, owners.full_name FROM animals RIGHT JOIN owners ON animals.owner_id=owners.id;
+
+/*// 4 How many animals are there per species?*/
+SELECT species.name, COUNT(animals.id) AS total_animals FROM animals INNER JOIN species ON animals.species_id = species.id GROUP BY species.name;
+
+/*// 5 List all Digimon owned by Jennifer Orwell.*/
+SELECT animals.name AS AnimalName, species.name AS SpeciesName, owners.full_name AS OwnerName FROM animals INNER JOIN species ON animals.species_id = species.id INNER JOIN owners ON animals.owner_id = owners.id WHERE owners.full_name='Jennifer Orwell' AND species.name= 'Digimon';
+
+
+/*// 6 List all animals owned by Dean Winchester that haven't tried to escape.*/
+SELECT animals.name, owners.full_name, animals.escape_attempts FROM animals INNER JOIN owners ON animals.owner_id=owners.id INNER JOIN species ON animals.species_id=species.id WHERE escape_attempts=0 AND owners.full_name='Dean Winchester';
+
+/*// 7 Who owns the most animals?*/
+SELECT owners.full_name, COUNT(animals.id) AS AnimalCount FROM owners INNER JOIN animals ON owners.id = animals.owner_id GROUP BY owners.id ORDER BY animal_count DESC LIMIT 1;
```

Comment:
- [ ] There's a problem with this query and it's also throwing an error in the resulting screenshot. Please make sure to fix it and also attached a screenshot of the working query.
  - You can update the query with the below code.
    `SELECT owners.full_name, COUNT(owners.full_name) FROM animals LEFT JOIN owners ON animals.species_id = 
     owners.id GROUP BY owners.full_name ORDER BY COUNT(owners.full_name) DESC;`

![image](https://user-images.githubusercontent.com/83803414/236011349-4ff5609c-3cdc-428c-a672-a3963f69cb03.png)